qiskit/utils/run_circuits.py

```diff
@@ -470,18 +476,31 @@ def run_circuits(
     Raises:
         QiskitError: Any error except for JobError raised by Qiskit Terra
     """
+    backend_version = getattr(backend, "version", None)
```

Comment:
Same here. Change `backend_version` to `backend_interface_version`.

Reply:
Done in: https://github.com/Qiskit/qiskit-terra/pull/7563/commits/363e8dd699bd099f54b18201c1217424e5383be8